doc-content/enterprise-only/openshift/operator-deploy-start-proc.adoc

```diff
@@ -0,0 +1,16 @@
+[id='operator-deploy-start-proc']
+= Starting the deployment of a {PRODUCT} environment using the Operator
+
+To start deploying a {PRODUCT} environment using the Operator, access the installer. The installer is deployed when you subscribe to the Operator.
+
+.Prerequisites
+
+* You have subscribed to the Operator. For instructions about subscribing to the Operator, see <<operator-subscribe-proc>>.
+
+.Procedure
+. In the {OPENSHIFT} Cluster Console menu, select *Networking* > *Routes*.
```

Comment:
Change this procedure by the dynamic link generated after the console ui deployment

Reply:
Done, I hope I got the procedure right, please double-check?